### PR TITLE
Plug and Play work

### PR DESCRIPTION
**Pull request by DavidHaywood**
new WORKING machines
----
Interactive TV Games 49-in-1 (PAL) [TeamEurope, David Haywood]
Teenage Mutant Ninja Turtles - Pocket Dream Console (France) [TeamEurope, David Haywood]

new NOT WORKING machines
----
Reactor MD (PAL) [TeamEurope, David Haywood]
Buzztime [Sean Riddle, David Haywood]

the SunPlus part of the Reactor boots and runs, the MD part I haven't attempted to emulate, the main menu seems to run in an added / unsupported video mode when I took a brief look.  As such, it gets marked as not working, as the SunPlus part is only meant to boot once you select it from the MD based main menu.

### CODE REVIEW

File: src/mame/drivers/spg2xx_pdc.cpp [hunk 1]
```diff
@@ -70,22 +79,29 @@ void spg2xx_pdc100_game_state::pdc100(machine_config &config)
 }
 
 ROM_START( pdc100 )
-	ROM_REGION( 0x8000000, "maincpu", ROMREGION_ERASE00 )
+	ROM_REGION( 0x4000000, "maincpu", ROMREGION_ERASE00 )
 	// only 1st half of this is used "Jumper resistor (0 ohm) that short A25 to ground"
 	// 2nd half just contains what seems to be random garbage
-	ROM_LOAD16_WORD_SWAP( "pdc100.bin", 0x000000, 0x8000000, CRC(57285b49) SHA1(cfb4be7877ec263d24063a004c56985db5c0f4e2) )
+	ROM_LOAD16_WORD_SWAP( "pdc100.bin", 0x000000, 0x4000000, CRC(57285b49) SHA1(cfb4be7877ec263d24063a004c56985db5c0f4e2) )
```
Comment (MoochMcGee) [inline src/mame/drivers/spg2xx_pdc.cpp]: Can you update the hashes please?

Comment (DavidHaywood) [reply src/mame/drivers/spg2xx_pdc.cpp]: the hashes don't need updating, the file hasn't changed.

Comment (MoochMcGee) [reply src/mame/drivers/spg2xx_pdc.cpp]: Except you changed the size of the file, and got rid of the second half filled with garbage, so of course the hashes are going to need updating.

Comment (MoochMcGee) [reply src/mame/drivers/spg2xx_pdc.cpp]: Except you changed the size and got rid of the entire second half, so the hashes will change anyway.

Comment (DavidHaywood) [reply src/mame/drivers/spg2xx_pdc.cpp]: No, I haven't changed the size of the file, read the whole code.

**Pull request merged**